composer.json

```diff
@@ -24,7 +24,7 @@
         "Laravel", "Backpack", "Backpack for Laravel", "Backpack Addon", "ThemeTabler"
     ],
     "require": {
-        "backpack/crud": "^6.1.12"
+        "backpack/crud": "^6.2"
```

Comment:
Heads-up @pxpm - we also need to bump the requirement for Backpack\CRUD here. Otherwise, if people use an updated version of `theme-tabler` and old version of `crud`, they get:

![CleanShot 2023-08-31 at 09 25 12](https://github.com/Laravel-Backpack/theme-tabler/assets/1032474/72955f88-9f34-4cdd-9bea-e49923db824d)